sonar-scanner-engine/src/main/java/org/sonar/batch/issue/IssueFilters.java

```diff
@@ -62,7 +62,7 @@ public boolean accept(String componentKey, BatchReport.Issue rawIssue) {
     return false;
   }
 
-  public boolean acceptDeprecated(String componentKey, BatchReport.Issue rawIssue) {
+  public boolean acceptDeprecated(String componentKey, ScannerReport.Issue rawIssue) {
     Issue issue = new DeprecatedIssueAdapterForFilter(project, rawIssue, componentKey);
     return new DeprecatedIssueFilterChain(deprecatedFilters).accept(issue);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "DeprecatedIssueFilterChain"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)